File: Professional WebBrowser - Step-By-Step - Part 1/[C#]-Professional WebBrowser - Step-By-Step - Part 1/C#/taapBrowser/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using taapBrowser.Forms;

namespace taapBrowser
{
    public partial class Form1 : Form
    {
        FrmBrowser browser;

        public Form1()
        {
            InitializeComponent();
            browser = new FrmBrowser();
            browser.Show(dockPanel);

        }
    }
}

File: Professional WebBrowser - Step-By-Step - Part 3/[VB]-Professional WebBrowser - Step-By-Step - Part 3/VB/taapBrowser/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports taapBrowser.Forms

Namespace taapBrowser
	Partial Public Class Form1
		Inherits Form

		Private browser As FrmBrowser

		Public Sub New()
			InitializeComponent()
			browser = New FrmBrowser()
			browser.Show(dockPanel)

		End Sub
	End Class
End Namespace
